packages/FormElement/stories/examples/ListBox.js

```diff
@@ -13,8 +13,9 @@ export default function ListBoxExample() {
       <FormElement>
         <Label>Form Label</Label>
         <Content>
-          {a11yProps => (
-            <ListBox {...a11yProps}>
+          {(a11yProps, refLabel) => (
```

Comment:
does this mean that all other elements except for the ListBox don't need the refLabel reference?

Reply:
what about 
`const { refLabel, ...moreAllyProps } = allyProps;`

Wouldn't that be enough and you wouldn't need to modify the FormElement component?

Reply:
> does this mean that all other elements except for the ListBox don't need the refLabel reference?

Yes, it's not a valid DOM attribute, and only being used by ListBox

> const { refLabel, ...moreAllyProps } = allyProps;

This can work, but all consumers will need to to this in all other type of input(input, textarea, checkbox...), except `ListBox`, if they don't do it, they'll get console.error in their app too

Reply:
ohh I see, ok, make sense to exclude the ListBox and force to destructure it in the component 👍